components/notification-sender-config/org.wso2.carbon.identity.notification.sender.tenant.config/src/main/java/org/wso2/carbon/identity/notification/sender/tenant/config/internal/NotificationSenderTenantConfigServiceDS.java

```diff
@@ -152,4 +156,27 @@ protected void setSMSProviderPayloadTemplateManager(SMSProviderPayloadTemplateMa
     protected void unsetSMSProviderPayloadTemplateManager(SMSProviderPayloadTemplateManager manager) {
         NotificationSenderTenantConfigDataHolder.getInstance().setSMSProviderPayloadTemplateManager(null);
     }
+
+    @Reference(name = "configuration.handler.tracker.service",
+            service = ChannelConfigurationHandler.class,
+            cardinality = ReferenceCardinality.MULTIPLE,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetConfigurationHandler")
+    protected void setConfigurationHandler(ChannelConfigurationHandler configurationHandler) {
+
+        if (log.isDebugEnabled()) {
```

Comment:
suggestion - no need to add a isDebugEnabled check is there are no arithmetic(string conct) operations inside that.